### DISCUSSION TITLE
why i cant get user articles

### DISCUSSION THREAD

**Discussion by TaherBarakat** [realworld-apps/realworld#1572] [Community]
the request is 
https://api.realworld.io/api/articles?offset=0&limit=20&author=qwerqwer
 response is
 {"articles":[],"articlesCount":0}
 and i have created many articles and requested them by the slug and it appears that they exist 
 whats the problem here?

**Comment by StevethecheeF** [comment]
I have a similar issue, where when I run the "/api/articles" request only articles from the logged-in user are returned.
I run the [node-express-prisma-v1-official-app](https://github.com/gothinkster/node-express-prisma-v1-official-app) locally, and I think I have read somewhere that this is expected for the demo but is there some kind of setting for local dev too?

**Comment by geromegrignon** [comment]
hi @TaherBarakat, as explained in the docs to avoid exposing malicious user created content to everyone, you can only see the content you created (and the boilerplate one). So if you are requesting the API directly, you need to provide a jwt token of the related user
